WL-2871

ColinHebert opened the pull request.

buckett commented on solr/impl/src/main/java/org/sakaiproject/search/solr/indexing/SolrTaskHandler.java:
```diff
@@ -341,6 +341,7 @@ private TaskHandlingException wrapException(Exception e, String message, Task po
         } else if (e instanceof SolrException
                 && (((SolrException) e).code() == SolrException.ErrorCode.SERVICE_UNAVAILABLE.code
                 || ((SolrException) e).code() == SolrException.ErrorCode.SERVER_ERROR.code
+                || ((SolrException) e).code() == SolrException.ErrorCode.NOT_FOUND.code
```
Is this unrelated to the NPE?

ColinHebert replied:
Details in 7ea7ecf

Commit:
Create subtree

Commit:
WL-2871

 Squashed commit of the following:

commit 50624108ff5c49d4f2b9cc9b6f12cc44d9da45f7
Merge: 14eab0d 4ee712c
Author: Colin Hebert <colin.hebert@it.ox.ac.uk>
Date:   Wed Jun 26 11:31:24 2013 +0100

    Merge branch 'master' into search-1.3.x

commit 4ee712c0e221426173ecd97fec4cd6c8363caad2
Author: Colin Hebert <colin.hebert@it.ox.ac.uk>
Date:   Tue Jun 25 15:08:52 2013 +0100

    Add the failed task to the logs of nestedTaskExceptions

commit b5aa47c7d58f0ef20bf8dc91d41cf64370a36210
Author: Colin Hebert <colin.hebert@it.ox.ac.uk>
Date:   Tue Jun 25 15:08:39 2013 +0100

    Remove unnecessary space

commit 14eab0da158fce7f15b6cba8823c8711e2516144
Merge: c4a0e9f 2946f36
Author: Colin Hebert <colin.hebert@it.ox.ac.uk>
Date:   Tue Jun 25 14:06:52 2013 +0100

    Merge branch 'master' into search-1.3.x

commit 2946f36626f033c1e53dcb1f13a56a57b6c9a620
Author: Colin Hebert <colin.hebert@it.ox.ac.uk>
Date:   Tue Jun 25 14:04:51 2013 +0100

    Do not escape nonchars from a null variable

commit 1b407f1736b9b70df7321909ed2640132d9b101b
Author: Colin Hebert <colin.hebert@it.ox.ac.uk>
Date:   Tue Jun 25 14:04:40 2013 +0100

    Consider 404 as a temporary issue

commit aa024a5d98b6554a9a4bd2732963f95c7b45640a
Author: Colin Hebert <colin.hebert@it.ox.ac.uk>
Date:   Tue Jun 25 14:04:25 2013 +0100

    Add documentation to the restlet shading

commit c4a0e9f6e7d879174e275c419ef281bebd93b62b
Merge: 27d5895 3fe3dae
Author: Colin Hebert <colin.hebert@it.ox.ac.uk>
Date:   Fri Jun 21 14:42:08 2013 +0100

    Merge branch 'master' into search-1.3.x

Commit:
Remove subtree original repo files

ColinHebert commented:
Code updated
